src/test/ui/rfc-2565-param-attrs/param-attrs-proc-macro-forbidden.rs

```diff
@@ -0,0 +1,18 @@
+// force-host
+// no-prefer-dynamic
+
+#![crate_type = "proc-macro"]
+#![feature(param_attrs)]
+
+extern crate proc_macro;
+
+use proc_macro::TokenStream;
+
+#[proc_macro_attribute]
+pub fn first(
+    #[bar] attr: TokenStream,
+    item: TokenStream
+) -> TokenStream {
+    //~^ ERROR it isn't possible to use #[proc_macro_attribute] and argument attributes
```

Comment:
This restriction is unnecessary and `fn forbid_proc_macro_attribute_with_arg_attrs` and this test can be removed.

The RFC meant that it's an error if `#[bar]` is a procedural macro, not that `#[proc_macro_attribute]` and param attrs cannot be used together on the same function.

Reply:
Removed